aws/resource_aws_elasticsearch_domain.go

```diff
@@ -117,6 +117,21 @@ func resourceAwsElasticSearchDomain() *schema.Resource {
 					},
 				},
 			},
+			"node_to_node_encryption": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
Maintainability note: We should try to avoid using `Computed: true` for container attributes and instead prefer to ignore the `"1" => "0"` difference (via `DiffSuppressFunc`). In this case `Computed: true` will mask when an Elasticsearch Domain is enabled for encryption, but not configured within Terraform (contrary to the resource documentation). We can fix this post-merge though.